client/src/app/app.module.ts

```diff
@@ -39,9 +39,7 @@ import { PostsComponent } from './components/posts/posts.component';
     FooterComponent,
     PaiComponent,
     FiltersComponent,
-    AboutComponent,
```

Comment:
Why you removed `AboutComponent`, `PostComponent`, and `PostsComponent` components?